quickstart/bin/bump-mongodb-migrate-image-digest.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
why is something specific for mongodb needed?

Reply:
Galoy backend defines and builds an image dedicated to running the migration scripts that we have. This script pins it to the sha that gets input. It's intended to be used from ci as part of the release process.
First the image must get built via https://ci.galoy.io/teams/dev/pipelines/galoy-app/jobs/build-migrate-edge-image/builds/1403 and then the ref will get bumped and committed back as part of the release.